concent_api/core/payments/backends/sci_backend.py

```diff
@@ -171,6 +171,21 @@ def call_on_confirmed_transaction(
     )
 
 
+def get_covered_additional_verification_costs(client_eth_address: str, payment_ts: int) -> list:
+    assert isinstance(client_eth_address, str) and len(client_eth_address) == ETHEREUM_ADDRESS_LENGTH
+    assert isinstance(payment_ts, int) and payment_ts >= 0
+
+    payment_interface: SCIImplementation = PaymentInterface()
+
+    last_block_before_payment = BlocksHelper(payment_interface).get_first_block_after(payment_ts).number
```

Comment:
Why is it called `last_block_before_payment` if you're taking the first block after?

Reply:
:+1: done